src/Microsoft.Owin.Security.Interop/Properties/AssemblyInfo.cs

```diff
@@ -1,15 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Reflection;
-using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: NeutralResourcesLanguage("en-us")]
-
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("a7922dd8-09f1-43e4-938b-cc523ea08898")]
```

Comment:
can this move to the csproj? We don't really need it, but I don't know what removing it would do.